org-code-javabuilder/lib/src/main/java/org/code/javabuilder/Util.java

```diff
@@ -2,18 +2,25 @@
 
```

Comment:
Should this be split into two separate util classes? The two sets of functions seem pretty different. (not a soft ask - curious if others have opinions)

Reply:
I debated this too. I saw two existing categories of util functions in this class before I added new ones: methods to deal with jars and a clean up method (`recursivelyClearDirectory`). Since the jar methods are used for compilation it made some sense to put the main method search functionality in here too, but I am open to suggestion. I think if we moved out the new methods it may also make sense to make a separate `JarUtil` class as well.

Reply:
Yeah, it seems like this file may have been created at a time when we didn't have a lot of util classes, but given that we now have several specialized util classes, might be worth renaming and/or breaking this up? I'd be okay with that being a separate task though.

Reply:
I'll do this as a follow up